packages/nodes-base/nodes/If/test/If.node.test.ts

```diff
@@ -0,0 +1,5 @@
+import { testWorkflows, getWorkflowFilenames } from '../../../test/nodes/Helpers';
+
+const workflows = getWorkflowFilenames(__dirname);
+
+describe('Test ItemLists Node', () => testWorkflows(workflows));
```

Comment:
You forgot to change the name of the test it's a bit confusing

Reply:
Fixed